BrowserKit/Sources/MenuKit/MenuRedesign/MenuCollectionView.swift

```diff
@@ -98,19 +88,29 @@ final class MenuCollectionView: UIView,
             return UICollectionViewCell()
         }
 
-        guard let section = menuData.first(where: { $0.isTopTabsSection }) else { return UICollectionViewCell() }
+        guard let section = menuData.first(where: { $0.isHorizontalTabsSection }) else { return UICollectionViewCell() }
         cell.configureCellWith(model: section.options[indexPath.row])
         if let theme { cell.applyTheme(theme: theme) }
         return cell
     }
 
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
         collectionView.deselectItem(at: indexPath, animated: false)
-        guard let section = menuData.first(where: { $0.isTopTabsSection }),
+        guard let section = menuData.first(where: { $0.isHorizontalTabsSection }),
```

Comment:
this check is done multiple times can be a computed var to avoid repeating the same code like
```
var horizontalTabsSection: MenuSection? {
    return menuData.first(where: { $0.isHorizontalTabsSection })
}
```